app.ts

```diff
@@ -56,12 +58,16 @@ export class App<
   notFound: Handler<Req, Res>
   attach: (req: Req, res: Res, next: NextFunction) => void
 
+  // this symbol tells if a custom error handler has been set
+  private readonly [hasSetCustomErrorHandler]: boolean
```

Comment:
use ESNext private properties instead of TS ones

Reply:
I changed it. But symbols are private I believe and I can't add a `#` in front since it's a runtime property.